README.md

```diff
@@ -45,6 +45,30 @@ Since semantic-release wraps the npm publish command, configuring provenance is
 Instead, provenance can be configured through the [other configuration options exposed by npm](https://docs.npmjs.com/generating-provenance-statements#using-third-party-package-publishing-tools).
 Provenance applies specifically to publishing, so our recommendation is to configure under `publishConfig` within the `package.json`.
 
+#### npm provenance on GitHub Actions
+
+For package provenance to be signed on the GitHub Actions CI the following permission is required
+to be enabled on the job:
+
+```yaml
+    permissions:
+      id-token: write # to enable use of OIDC for npm provenance
+```
+
+It's worth noting that if you are using semantic-release to its fullest with a GitHub release, GitHub comments
+and other features then more permissions are required to be enabled on this job:
```

Comment:
I think it could be worth linking to where these details are defined in the docs for our github plugin. what do you think?
```suggestion
It's worth noting that if you are using semantic-release to its fullest with a GitHub release, GitHub comments,
and other features, then [more permissions are required](https://github.com/semantic-release/github#github-authentication) to be enabled on this job:
```

Reply:
Yep, looks great!